tidy3d/components/data/data_array.py

```diff
@@ -71,16 +71,23 @@ class DataArray(xr.DataArray):
     _data_attrs: Dict[str, str] = {}
 
     def __init__(self, data, *args, **kwargs):
-        """Initialize ``DataArray``."""
```

Comment:
put docstring back?

Reply:
Oops. Although, to be fair, this is like
```python
# initialize x
x = 0
```